manifests/plugin/filter/chain.pp

```diff
@@ -1,8 +1,8 @@
 # https://collectd.org/wiki/index.php/Chains
 define collectd::plugin::filter::chain (
-  $ensure         = 'present',
-  $target         = undef,
-  $target_options = undef,
+  Enum['present', 'absent'] $ensure                                                           = 'present',
+  Optional[Enum['notification', 'replace', 'set', 'return', 'stop', 'write', 'jump']] $target = undef,
```

Comment:
the line gets pretty long, should we create a custom datatype for this?

Reply:
Yes